source/content/localdev.md

```diff
@@ -116,35 +130,39 @@ The steps in this section should only be used as a last resort. This resets Loca
 Localdev does not connect to [Multidev](/multidev/) environments or allow direct database access.
 
 ## Changelog
-
-### 0.6.0-beta9
-Dec. 4, 2019
-
-#### New features
-- Added macOS Catalina Support!
-- Added a `beta` release channel
-- Added ability to toggle `edge` service [#24](https://github.com/pantheon-systems/localdev-issues/issues/24)
-- Added ability to rebuild a given app
-- Added basic online/offline detection
-- Added support for `phpmyadmin` as toggleable service
-- Added surfacing of release notes
-- Added option to automatically quit Docker on Localdev quit
-- Improvements
-- Improved cancellation of app initialization
-- Improved application state detection
-- Improved shutdown and quit
-- Improved process handling and management
-- Improved revoked token detection and mitigation
-- Improved application destruction UX
-- Improved Docker state management
-- Improved error handling and logging
-- Improved "botched state" cleanup
-- Improved progress indicators
-
-#### Bug fixes
-- Fixed various initialization bugs
-- Fixed various "app is stuck" bugs
-- Fixed bug causing pre-0.6.0 installs to not be detected
-
-#### Notes
-- Going forward Localdev will manage its own Lando configuration
+See the [Localdev Changelog file](https://pantheon-localdev.s3.amazonaws.com/changelog.yml) for the full version history.
```

Comment:
Why are we pointing to this URL instead of just having changelog in this documentation page? I think it's preferable to have CHANGELOG at pantheon.io/docs rather than something that includes amazonaws

Reply:
Actually, we're working on automating that in [PR 5456](https://github.com/pantheon-systems/documentation/pull/5453). It's here in static format until that automation is complete.